src/components/button/button.css

```diff
@@ -0,0 +1,105 @@
+@import url('/src/components/utils.css');
+
+.container-row{
+    justify-content: space-evenly;
+    
+}
+.btn{
+    height : 5rem ;
+    width : 13rem ;
+    font-size : 1.5rem ;
+    font-weight : bold  ;
+    border-radius: 1rem ;
+    border : none ;
+    cursor: pointer;
+}
+.btn-primary{
+    color : var(--white-color) ;
+    background-color: var(--ca-primary-color);
+}
+.btn-primary:hover{
+    background-color: var(--ca-primary-color-lighter);
+    font-size : 1.8rem ;
+}
+.btn-success{
+    color : var(--white-color);
+    background-color : var(--success-text);
+}
+.btn-success:hover{
+    background-color: var(--online-color);
+    font-size : 1.8rem ;
```

Comment:
One suggestion Instead of increasing font-size you can use transform property.

Reply:
will look in to that soon